src/main/java/org/jabref/gui/util/DefaultFileUpdateMonitor.java

```diff
@@ -29,11 +31,14 @@
 
     private final Multimap<Path, FileUpdateListener> listeners = ArrayListMultimap.create(20, 4);
     private WatchService watcher;
+    private Optional<JabRefException> filesystemMonitorFailure;
```

Comment:
I think, it's OK to store the exception (for future work).

Reply:
What do you mean? I am not sure what the code change is that you are requesting.

Reply:
This was more a comment to a second reviewer.

Reply:
https://github.com/JabRef/user-documentation/pull/286